### PR TITLE
[file organization] Rename folder names to allow markers to work again

### LINKED ISSUE #1668: [pytest] Unable to run pytest by marker
**Description**

We have some tests marked with markers:
```
pytest --markers
@pytest.mark.acl: ACL tests
@pytest.mark.bsl: BSL tests
@pytest.mark.reboot: tests which perform SONiC reboot
@pytest.mark.port_toggle: tests which toggle ports
```

Trying to run a multiple pytests by marker I get an error during test collection ( see below ).
From traceback it is not clear what went wrong. The [`-m` pytest option](https://docs.pytest.org/en/latest/example/markers.html) worked a couple of month ago ( tested with 8fd453aa805d25c114a6f1fb6e7906a48620a88a).

**Steps to reproduce the issue:**
1. pytest -m {acl|bsl|reboot|port_toggle} 
2.
3.

**Describe the results you received:**
```
pytest -m bsl  --module-path=../ansible/library --inventory=../ansible/inventory --testbed_file=../ansible/testbed.csv --testbed=
${SWITCH}-${TOPO} --host-pattern=${SWITCH}-${TOPO} --show-capture=no                                                                                                             
============================================================================== test session starts ==============================================================================
platform linux2 -- Python 2.7.12, pytest-4.6.9, py-1.8.1, pluggy-0.13.1
ansible: 2.8.7
rootdir: /mykolaf/sonic-mgmt/tests, inifile: pytest.ini
plugins: ansible-2.2.2
collected 0 items / 1 errors                                                                                                                                                    
WARNING: No route found for IPv6 destination :: (no default route?)

==================================================================================== ERRORS =====================================================================================
_________________________________________________________________________ ERROR collecting test session _________________________________________________________________________
/usr/local/lib/python2.7/dist-packages/py/_path/common.py:383: in visit
    for x in Visitor(fil, rec, ignore, bf, sort).gen(self):
/usr/local/lib/python2.7/dist-packages/py/_path/common.py:425: in gen
    if p.check(dir=1) and (rec is None or rec(p))])
/usr/local/lib/python2.7/dist-packages/_pytest/main.py:667: in _recurse
    ihook = self.gethookproxy(dirpath)
/usr/local/lib/python2.7/dist-packages/_pytest/main.py:482: in gethookproxy
    my_conftestmodules = pm._getconftestmodules(fspath)
/usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:424: in _getconftestmodules
    mod = self._importconftest(conftestpath.realpath())
/usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:463: in _importconftest
    raise ConftestImportFailure(conftestpath, sys.exc_info())
E   ConftestImportFailure: (local('/mykolaf/sonic-mgmt/tests/arp/conftest.py'), (<type 'exceptions.ValueError'>, ValueError('Attempted relative import in non-package',), <traceb
ack object at 0x7f76bb0813f8>))
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================ 1 error in 0.41 seconds ============================================================================
```

**Describe the results you expected:**

Tests marked with `@pytest.mark.bsl` selected and executed

**Additional information you deem important:**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

### PR DESCRIPTION
**Pull request by yxieca**
Summary:
Fixes #1668 

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### How did you do it?
pytest has an issue dealing with sub-folders with the same name and being
included from. In this case, the problem was with several components has
'args' as sub-folder to hold argument parsing utilities.

This issue was initially caught when we were tryign to run the whole test
suite. Attempt was made to address this issue. However there are more in
pytest that cannot tolerate same name sub-folders.

Change 'args' to '\<feature\>_args' and change conftest.py accordingly.

This PR also reverted PR #1642 

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

#### How did you verify/test it?
py.test ... ... -m bsl --collect-only

Without fix, nothing was collected. With the fix:

collected 777 items / 1 errors / 766 deselected / 10 selected                                                                                                                                                                           
\<Module tests/fdb/test_fdb.py\>
  \<Function test_fdb[ethernet]\>
  \<Function test_fdb[arp_request]\>
  \<Function test_fdb[arp_reply]\>
\<Module tests/snmp/test_snmp_cpu.py\>
  \<Function test_snmp_cpu\>
\<Module tests/snmp/test_snmp_interfaces.py\>
  \<Function test_snmp_interfaces\>
\<Module tests/snmp/test_snmp_lldp.py\>
  \<Function test_snmp_lldp\>
\<Module tests/snmp/test_snmp_psu.py\>
  \<Function test_snmp_numpsu\>
  \<Function test_snmp_psu_status\>
\<Module tests/vlan/test_vlan.py\>
  \<Function test_vlan_tc1_send_untagged\>
  \<Function test_vlan_tc2_send_tagged\>
  \<Function test_vlan_tc3_send_invalid_vid\>

### CODE REVIEW

File: tests/common/plugins/sanity_check/__init__.py [hunk 1]
```diff
@@ -13,6 +13,14 @@
 logger = logging.getLogger(__name__)
 
 
+def pytest_addoption(parser):
```
Comment (daall) [inline tests/common/plugins/sanity_check/__init__.py:16]: I get the rest of the changes, but why do these need to be moved?

Comment (yxieca) [reply tests/common/plugins/sanity_check/__init__.py:16]: This is reverting #1642.

Comment (daall) [reply tests/common/plugins/sanity_check/__init__.py:16]: Ok, this seems like an odd place for these, let's revisit it later. :P

**Comment by tahmed-dev** [comment]
I suspect there is something else going on as pytest recommends having conftest in a package scope (adding __init__).
https://docs.pytest.org/en/2.7.3/plugins.html?highlight=conftest

**Pull request merged**